File: Plugins/UnrealDotNet/Source/UnrealDotNetRuntime/Private/Generate/Manage/ManageShapeComponent.cpp
#include "UnrealDotNetRuntime.h"
#include "Generate/Manage/ManageShapeComponent.h"

PRAGMA_DISABLE_DEPRECATION_WARNINGS

// Source file C:\Users\vladi\Desktop\Engine\Source\Runtime\Components\ShapeComponent.h:24

void UManageShapeComponent::UpdateBodySetup()
{
	Super::UpdateBodySetup();
	if(bIsManageAttach) UCoreShell::InvokeInObject(this, "UpdateBodySetup");
}

PRAGMA_ENABLE_DEPRECATION_WARNINGS

File: Plugins/UnrealDotNet/Source/NetCoreProject/UnrealEngine/Generate/Manage/ManageShapeComponent.cs
using System;
using System.Runtime.InteropServices;

// Source file C:\Users\vladi\Desktop\Engine\Source\Runtime\Components\ShapeComponent.h:24

namespace UnrealEngine
{
	public partial class ManageShapeComponent : UShapeComponent
	{
		public ManageShapeComponent(IntPtr Adress)
			: base(Adress)
		{
		}

		
		/// <summary>
		/// <para>Update the body setup parameters based on shape information </para>
		/// </summary>
		public override void UpdateBodySetup() { }
		
		public static implicit operator IntPtr(ManageShapeComponent Self)
		{
			return Self.NativePointer;
		}

		public static implicit operator ManageShapeComponent(ObjectPointerDescription PtrDesc)
		{
			return NativeManager.GetWrapper<ManageShapeComponent>(PtrDesc);
		}}}
